src/lib/moderation.ts

```diff
@@ -18,16 +18,23 @@ export function describeModerationCause(
   }
   if (cause.type === 'blocking') {
     return {
-      name: 'Blocked User',
+      name: 'Blocked Author',
       description: 'You have blocked this user. You cannot view their content.',
     }
   }
   if (cause.type === 'blocked-by') {
     return {
-      name: 'Blocking You',
+      name: 'Blocked',
```

Comment:
This might be confusing. Since 'User Blocks You' is used in `ModerationDetails.tsx` I suggest we keep this as 'Blocked you'